### PR TITLE
Mark "split large buffers" test flaky

### PR DESCRIPTION
**Pull request by rossabaker**
Alternative to #771.  Better discussion there, but this reflects that we don't yet understand it, but also lets us move forward.

### CODE REVIEW

**Comment by rossabaker** [comment]
Uh... does `.flaky` not do what I remember?

File: core/src/test/scala/org/http4s/blaze/pipeline/stages/SSLStageSuite.scala [hunk 1]
```diff
@@ -61,7 +61,9 @@ class SSLStageSuite extends BlazeTestSuite {
       } yield ()
     }
 
-    test(testSuitePrefix + " should split large buffers") {
+    test((testSuitePrefix + " should split large buffers").flaky) {
```
Comment (danicheg) [inline core/src/test/scala/org/http4s/blaze/pipeline/stages/SSLStageSuite.scala:64]: It makes me whaaaat
```scala
test(testSuitePrefix + " should split large buffers".flaky) {} // compiles but doesn't work properly
```

**Comment by danicheg** [comment]
@rossabaker hope you don't mind that I dropped my few lines here

File: core/src/test/scala/org/http4s/blaze/pipeline/stages/SSLStageSuite.scala (hunk 1 above)
Comment (danicheg) [reply core/src/test/scala/org/http4s/blaze/pipeline/stages/SSLStageSuite.scala:64]: though nothing new, silly type-weakness of string concatenation

Comment (rossabaker) [reply core/src/test/scala/org/http4s/blaze/pipeline/stages/SSLStageSuite.scala:64]: Oh, good find.  I always thought it was odd that operator went on the String in the first place.

**Pull request merged**